tools/spell_checker/dictionary.txt

```diff
@@ -3177,6 +3178,7 @@ kPa
 krabgek
 krabgeks
 Krag
+Krampouezh
```

Comment:
```suggestion
krampouezh
```
It is not a proper noun so it should be uncapitalized in the dictionary.

Reply:
I'm a bit new to this dictionary stuff, thanks